.github/ISSUE_TEMPLATE/technical_story.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Technical story
+about: Create a new technical story for this project
+title: "[Name this TS as it appears in the sprint goal]"
```

Comment:
how about technical enabler instead of technical story?
I also don't think it makes sense to mention the sprint goal